python/nano/src/bigdl/nano/deps/ipex/ipex_inference_model.py

```diff
@@ -203,16 +206,25 @@ def status(self):
         return status
 
     @staticmethod
-    def _load(path, model, inplace=False):
+    def _load(path, model, input_sample=None, inplace=False):
         status = PytorchIPEXJITModel._load_status(path)
         checkpoint_path = path / status['checkpoint']
         if status["use_jit"]:
-            if status["use_ipex"]:
-                import intel_extension_for_pytorch as ipex
-            model = torch.jit.load(checkpoint_path)
-            model.eval()
-            model = torch.jit.freeze(model)
-            from_load = True
+            if status['compression'] == "bf16":
+                state_dict = torch.load(checkpoint_path, map_location='cpu')
+                if status['compression'] == "bf16":
+                    state_dict = transform_state_dict_to_dtype(state_dict, dtype="fp32")
```

Comment:
Can we convert loaded state dict to 'fp32' directly ? Maybe it was not 'fp32' before saving.

Reply:
Now it seems we only support fp32 for JIT? maybe we can update this code when we actually meet new use case?
Or obtain the dtype of model by something like `list(model.parameters())[0].dtype`

Reply:
yeah, this might be a problem, we should have a better logic in `transform_state_dict_to_dtype`. Currently we use `is_floating_point` to decide which parameter needs to be revised.